crypto/x509/x509name.c

```diff
@@ -87,8 +87,8 @@ X509_NAME_ENTRY *X509_NAME_get_entry(const X509_NAME *name, int loc)
     if (name == NULL || sk_X509_NAME_ENTRY_num(name->entries) <= loc
         || loc < 0)
         return (NULL);
```

Comment:
well if this PR wasn't all about nits and style, then I wouldn't point this out, but ... extra paren's.